timed/employment/admin.py

```diff
@@ -146,77 +144,77 @@ class AbsenceCreditInline(admin.TabularInline):
 class UserAdmin(UserAdmin):
     """Timed specific user admin."""
 
-    inlines = [
+    inlines = (
         SupervisorInline,
         SuperviseeInline,
         EmploymentInline,
         OvertimeCreditInline,
         AbsenceCreditInline,
-    ]
+    )
     list_display = ("username", "first_name", "last_name", "is_staff", "is_active")
```

Comment:
this unchanged line is worth mentioning in the context of https://github.com/adfinis/timed-backend/pull/1049/files#r1568378503

we should be consistent, either tuple all the things or list all the things

tbh, i'm not completely sure what django does with these under the hood, but their semantics don't seem to have any impact on the correctness of the resulting app

Reply:
Nope, and as you noted elsewhere, DRF isn't too consistent either. I think they're doing tuples to avoid accidental mutations, but I try to avoid tuples in the general case as they can be a source of nasty errors, especially in the single-value kind - forget the trailing comma, and the braces become just braces, and if the containing value happens to be an iterable, and the "user" of the value happens to not explicitly type-check, you suddenly have  "f", "o", "o" instead of "foo" somewhere

Reply:
I didn't check DRF itself given that this is a Django Admin module. From the looks of it and given it's age the `django.contrib.admin` isn't where i'm looking for inspiration on how to write pythonic code.

Reply:
The "f". "o", "o" bug is just as much taken care of by ruff format btw:

```console
$ cat test.py 
v = ("foo")
hairmare@lina:~/Documents/git.repos$ ruff format test.py 
1 file reformatted
hairmare@lina:~/Documents/git.repos$ cat test.py 
v = "foo"
$ vim test.py # hack hack hack
$ cat test.py 
v = ("foo",)
hairmare@lina:~/Documents/git.repos$ ruff format test.py 
1 file left unchanged
```

together with some simple typing as per #1054 i don't think this bug can be triggered w/o being caught in review